File: OPENAPI-REST-API/openapi-client/php-dt/src/App/DTO/AccountInfoDataCc.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\PhpAttribute as DTA;

/**
 * Credit card information stored on the account.
 */
class AccountInfoDataCc
{
    #[DTA\Data(field: "cc", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $cc = null;

    #[DTA\Data(field: "cc_exp", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $cc_exp = null;

    #[DTA\Data(field: "name", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $name = null;

    #[DTA\Data(field: "address", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $address = null;

    #[DTA\Data(field: "city", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $city = null;

    #[DTA\Data(field: "state", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $state = null;

    #[DTA\Data(field: "zip", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $zip = null;

    #[DTA\Data(field: "country", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $country = null;

    #[DTA\Data(field: "maxmind_riskscore", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $maxmind_riskscore = null;

    #[DTA\Data(field: "maxmind", nullable: true)]
    #[DTA\Strategy("Object", ["type" => \App\DTO\AccountInfoMaxMindResponse::class])]
    #[DTA\Validator("TypeCompliant", ["type" => \App\DTO\AccountInfoMaxMindResponse::class])]
    public \App\DTO\AccountInfoMaxMindResponse|null $maxmind = null;

    #[DTA\Data(field: "verified", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "bool"])]
    public bool|null $verified = null;

}

File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/AccountInfoDataCc.scala
package io.swagger.server.model


/**
 * = AccountInfoDataCc =
 *
 * Credit card information stored on the account.
 *
 * @param cc 
 * @param cc_exp 
 * @param name 
 * @param address 
 * @param city 
 * @param state 
 * @param zip 
 * @param country 
 * @param maxmind_riskscore 
 * @param maxmind 
 * @param verified 
 */
case class AccountInfoDataCc (
  cc: Option[String],
  cc_exp: Option[String],
  name: Option[String],
  address: Option[String],
  city: Option[String],
  state: Option[String],
  zip: Option[String],
  country: Option[String],
  maxmind_riskscore: Option[String],
  maxmind: Option[AccountInfoMaxMindResponse],
  verified: Option[Boolean]
)

